spec/re_captcha/configuration_spec.rb

```diff
@@ -26,20 +26,86 @@
     end
 
     describe '#valid?' do
+      let(:errors) { double('Array', :<< => nil) }
+
+      it 'resets errors' do
+        expect(instance).to receive(:reset_errors)
+        instance.valid?
+      end
+
+      it 'sets errors' do
+        expect(instance).to receive(:set_errors)
```

Comment:
same here - just test the expected results - that the instance has errors, not that it receives some private method